terraform/bigquery.tf

```diff
@@ -436,8 +436,10 @@ resource "google_bigquery_table" "unique_events_view" {
       JSON_VALUE(payload, "$.repository.visibility") repository_visibility,
       JSON_VALUE(payload, "$.sender.login") sender,
       SAFE_CAST(JSON_QUERY(payload, "$.sender.id") AS INT64) sender_id,
-    FROM
-      `${google_bigquery_dataset.default.dataset_id}.${google_bigquery_table.events_table.table_id}`
+    FROM (
```

Comment:
I plugged this query into the UI and its throwing errors on the above JSON_QUERY functions. Should they be changed to JSON_VALUE?

Reply:
@verbanicm I was able to run the query successfully. Maybe you had a type when you copied?

Here is what I ran in the UI:
```
SELECT
  delivery_id,
  signature,
  received,
  event,
  payload,
  JSON_VALUE(payload, "$.organization.login") ORGANIZATION,
  SAFE_CAST(JSON_VALUE(payload, "$.organization.id") AS INT64) organization_id,
  JSON_VALUE(payload, "$.repository.full_name") repository_full_name,
  SAFE_CAST(JSON_QUERY(payload, "$.repository.id") AS INT64) repository_id,
  JSON_VALUE(payload, "$.repository.name") repository,
  JSON_VALUE(payload, "$.repository.visibility") repository_visibility,
  JSON_VALUE(payload, "$.sender.login") sender,
  SAFE_CAST(JSON_QUERY(payload, "$.sender.id") AS INT64) sender_id,
FROM (
  SELECT
    ROW_NUMBER() OVER (PARTITION BY delivery_id ORDER BY received DESC) AS row_id,
    *
  FROM
    `github-metrics-aggreg-p-109ae8.github_metrics.events`)
WHERE
  row_id = 1;
  ```

Reply:
I probably should have also changed that one to JSON_VALUE, but I already changed the others. On this table it doesn't matter, since payload is a string and not a json type, and JSON_QUERY has different behavior than JSON_VALUE for JSON types (which is super annoying). 

SAFE_CAST(JSON_QUERY( and SAFE_CAST(JSON_VALUE are the same for the events table but not the raw events due to this

Reply:
Fixed by merging in #154